include/fat.h

```diff
@@ -39,88 +39,90 @@ enum fat_cache_type {
     FAT_CACHE_MAX
 };
 
+// clang-format off
```

Comment:
why?

Reply:
the Alignas makes `data` align weirdly, adding a `;` makes it work properly but it generates an error when building it